applications/semaphore/values-usdfdev.yaml

```diff
@@ -1,9 +1,12 @@
 semaphore:
-  config:
-    phalanx_env: "usdfdev"
-    github_app_id: "131502"
-    enable_github_app: "True"
   ingress:
     enabled: true
     annotations:
-      kubernetes.io/ingress.class: nginx
+      kubernetes.io/ingress.class: nginx  
+pull-secret:
+  enabled: true
+  path: secret/rubin/usdf-rsp-dev/semaphore
```

Comment:
This is for a Kubernetes pull-secret. You don't want to set this (we should probably delete it from the configuration entirely). I think Semaphore always reads its normal secret from Vault.